moshi_mlx/moshi_mlx/local.py

```diff
@@ -371,10 +371,14 @@ def main(printer: AnyPrinter):
     printer.log("info", "All done!")
 
 
-if __name__ == "__main__":
+def _main():
```

Comment:
final nit: for consistency with the other module, can you call this `main` ?

Reply:
It would conflict with the existing `main`, but I can move the `printer` init into the `main` function, I think that makes sense, but I'm not sure if there was a reason for it not to be in the first place.